draft-ietf-babel-information-model.md

```diff
@@ -648,42 +615,78 @@ babel-route-selected:
   is being advertised).
 
 
-# Common Objects
-
-## Definition of babel-credential-obj
+## Definition of babel-security-obj
 
 ~~~~
-     object {
-          credentials          ro babel-cred;
-    } babel-credential-obj;
+  object {
+       string                ro babel-security-mechanism

```

Comment:
Again, if this is an enumeration of type babel-security-supported, should this not be an enum?

Reply:
There is no "enum" type defined.